### DISCUSSION TITLE
Homebridge Characteristic Warning

### DISCUSSION THREAD

**Discussion by JackWAllen94** [milo526/homebridge-tuya-web#367] [Q&A]
Hey,

I've had the error message shown in the attached screenshot appear in Homebridge, the link Homebridge provided suggested I should raise the issue with the developer.

Link in the screenshot https://github.com/homebridge/homebridge/wiki/Characteristic-Warnings
<img width="1035" alt="Screenshot 2021-11-28 at 22 14 15" src="https://user-images.githubusercontent.com/53379246/143788229-0d4c0013-5e9a-4dd4-8d65-ff20517c058c.png">

**Comment by yayfrogs** [comment]
I’ve recently started having this issue. Is there a fix?
